cypress/integration/MarkdownEditorSpecialCharacter.spec.ts

```diff
@@ -47,7 +48,7 @@ describe('Markdown Editor / Insert Special Character Dialog', () => {
   }
 
   function insertSpecialCharacter(char: string) {
-    selectors.getSpecialCharacterButtons().findByText(char).click();
```

Comment:
It seems chaining is discouraged and it doesn't work as expected: https://github.com/testing-library/cypress-testing-library/issues/132